capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js

```diff
@@ -18,7 +18,74 @@
 function createMap() { // eslint-disable-line no-unused-vars
   const map = new google.maps.Map(
       document.getElementById('map'),
-      {center: {lat: 39.828502, lng: -98.579512}, zoom: 4});
+      {center: {lat: 39.828502, lng: -98.579512},
+        zoom: 4,
+      },
+  );
+
+  // Set custom color scheme for the map.
+  const styles =
+    [
+      {
+        'featureType': 'administrative',
+        'elementType': 'labels.text.fill',
+        'stylers': [
+          {
+            'color': '#000000',
+          },
+        ],
+      },
+      {
+        'featureType': 'landscape.man_made',
+        'elementType': 'geometry.fill',
+        'stylers': [
+          {
+            'color': '#b6e59e',
+          },
+          {
+            'visibility': 'on',
+          },
+        ],
+      },
+      {
+        'featureType': 'landscape.natural.landcover',
+        'elementType': 'geometry.stroke',
+        'stylers': [
+          {
+            'color': '#bddda7',
+          },
+        ],
+      },
+      {
+        'featureType': 'road',
+        'elementType': 'geometry.fill',
+        'stylers': [
+          {
+            'color': '#f3d36b',
+          },
+        ],
+      },
+      {
+        'featureType': 'road.highway',
+        'elementType': 'geometry.fill',
+        'stylers': [
+          {
+            'color': '#ffa662',
+          },
+        ],
+      },
+      {
+        'featureType': 'water',
+        'elementType': 'geometry.fill',
+        'stylers': [
+          {
+            'color': '#75cff0',
+          },
+        ],
+      },
+    ];
+
+  map.setOptions({styles: styles});
```

Comment:
nit: Shorthand this to `{styles}`. [ref](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer)

Reply:
Fixed! @Leoism